tutorials/themes/tutorials-theme/navbar.html

```diff
@@ -39,9 +39,10 @@ <h2 class="card-title text-center"><a href="{{ pathto('examples') }}" class="tex
 </div>
 
 {% else %}
-
 <nav class="navbar navbar-expand-lg bg-navbar">
-  <a class="navbar-brand" href="/astropy-tutorials"style="padding: 0px 5px;"><img src="{{ pathto('_static/' + 'astropy_logo.png') }}" style="height:38px;"></a>
```

Comment:
Instead of hard-linking to `astropy.org/astropy-tutorials`, I would instead recommend changing this to:

```html
<a class="navbar-brand" href="/astropy-tutorials"style="padding: 0px 5px;">
<img src="{{ pathto('_static/' + 'astropy_logo.png', 1) }}" style="height:38px;">
</a>
```

[I just learned here](https://www.sphinx-doc.org/en/master/templating.html) that `pathto(file)` links to an HTML document (so it was breaking the link), but `pathto(file, 1)` will link to a static file (i.e. an image, as we want here), so this is what we want.

Reply:
Sure, that would be better...